.github/.jazzy.yaml

```diff
@@ -7,4 +7,47 @@ module: SwiftCurrent
 author_url: https://github.com/wwt/SwiftCurrent
 github_url: https://github.com/wwt/SwiftCurrent
 sourcekitten_sourcefile: swiftcurrent-docs.json,swiftcurrentuikit-docs.json,swiftcurrent-swiftui-docs.json
-author: WWT and Tyler Thompson
\ No newline at end of file
+author: WWT and Tyler Thompson
+documentation: .github/guides/*.md
+abstract: .github/abstract/*.md
+custom_categories:
+- name: How to use SwiftCurrent with SwiftUI
+  children:
+    - Getting Started with SwiftUI
+    - Working with NavigationView
+- name: How to use SwiftCurrent with UIKit
+  children:
+    - Storyboards Intro
+    - Programmatic UIKit Intro
+- name: Creating Workflows
+  children:
+    - FlowRepresentable
+    - PassthroughFlowRepresentable
+    - Workflow
+- name: Creating Workflows in SwiftUI
+  children:
+    - View
+    - ViewControllerWrapper
+    - WorkflowItem
+    - WorkflowLauncher
+- name: Creating Workflows in UIKit
```

Comment:
I like how there is a little preamble for those who click `How to use SwiftCurrent with SwiftUI` and I think we may want something here as well.

Reply:
Agreed, think we need a documentation follow-up.

Reply:
I agree, and also, I can't GitHub.  I meant to put this on `Creating Workflows in SwiftUI` specifically because this did some shuffling of what you care about and how it can be used.